server/protocols/jmap/src/main/java/org/apache/james/jmap/JMAPRoutes.java

```diff
@@ -57,4 +55,9 @@
         logger().warn("Unauthorized", e);
         return response.status(UNAUTHORIZED).send();
     }
+
+    default Mono<Void> handleForbidden(HttpServerResponse response) {
+        logger().error("Not implemented yet");
```

Comment:
then it should be ` default Mono<Void> handleNotImplemented(HttpServerResponse response)`